src/Day2/StopWatch.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Created by TTND on 12/27/2016.
+ * Create a stop watch component. called StopWatch This component should have a button and a timer inside it. On click of the button, the timer should start and should display the time passed (since the click of the button) every second. On click of the button again, the timer should stop and the displayed time should freeze.
+
+ TIP: componentWillUnmount should be used
+ */
+import React from 'react';
+
+
+export default class StopWatch  extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state= {
+            buttonVal: 'Start',
+            time: 0
+        };
+        this.handleClick =this.handleClick.bind(this);
+
+    }
+
+    componentDidMount() {
+
+        if(this.state.buttonVal =="Start") {
+            this.timerID = setInterval(
+                () => this.changeTime(),
+                0.001
```

Comment:
This does not look like `one second`. I would assume one second would be `1000ms` instead of `0.001`